examples/soap-calculator/test/acceptance/application.acceptance.ts

```diff
@@ -0,0 +1,79 @@
+import {supertest} from '@loopback/testlab';
+import {SoapCalculatorApplication} from '../../src/application';
+import {expect} from '@loopback/testlab';
+
+describe('Application', function() {
+  let app: SoapCalculatorApplication;
+  let client: supertest.SuperTest<supertest.Test>;
+
+  // Increase the timeout to accommodate slow network connections
+  // tslint:disable-next-line:no-invalid-this
+  this.timeout(30000);
+
+  before(givenAnApplication);
+
+  before(async () => {
+    await app.boot();
+    await app.start();
+  });
+
+  before(() => {
+    client = supertest(app.restServer.url);
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  it('rejects division by zero with 412 error', async () => {
+    await client.get('/divide/20/0').expect(412);
+  });
+
+  it('returns 404 when the "/divide" is missing required args', async () => {
+    await client.get('/divide/').expect(404);
+  });
+
+  it('returns 404 when the "/multiply" is missing required args', async () => {
+    await client.get('/multiply/').expect(404);
+  });
+
+  it('returns 404 when the "/add" is missing required args', async () => {
+    await client.get('/add/').expect(404);
+  });
+
+  it('returns 404 when the "/subtract" is missing required args', async () => {
+    await client.get('/subtract/').expect(404);
+  });
+
+  it('divides two numbers', async () => {
+    const response = await client.get('/divide/50/2').expect(200);
+    const answer = {result: {DivideResult: 25}};
+    expect(response.body).to.containDeep(answer);
+  });
+
+  it('adds two numbers', async () => {
+    const response = await client.get('/add/25/25').expect(200);
+    const answer = {result: {AddResult: 50}};
+    expect(response.body).to.containDeep(answer);
+  });
+
+  it('multiplies two numbers', async () => {
+    const response = await client.get('/multiply/25/3').expect(200);
+    const answer = {result: {MultiplyResult: 75}};
+    expect(response.body).to.containDeep(answer);
+  });
+
+  it('subtracts two numbers', async () => {
+    const response = await client.get('/subtract/100/25').expect(200);
+    const answer = {result: {SubtractResult: 75}};
+    expect(response.body).to.containDeep(answer);
+  });
+
+  function givenAnApplication() {
+    app = new SoapCalculatorApplication({
+      rest: {
+        port: 0,
```

Comment:
Let's try add `host: '127.0.0.1` here to fix Travis. (Just a guess).

Reply:
Thanks @virkt25 , I will try this.

Reply:
🥇 Yes!, the hots: '127.0.0.1' worked on Travis. Thank you @virkt25 .